### ISSUE TITLE
How to write YOLO or U-Net Deep Learning code which is integrated with RasterFrames? 

### ISSUE THREAD

**Issue by JenniferYingyiWu2020** [Esri/raster-functions#72]
Hi,
        I have read the RasterFrames supervised machine learning on the page of "https://rasterframes.io/supervised-learning.html", now I would like to implement Deep Learning similar to “supervised machine learning”, such as "YOLO object detection" and "U-Net semantic segmentation". 
        However, I am so confused about how to write YOLO or U-Net Deep Learning code  which is integrated with RasterFrames? So, would you pls help to give me some suggestions? Many Thanks!

**Comment by shenganzhan** [comment]
Hi, Are you trying to write a ESRI raster function to utilize RasterFrames to do deep learning?

**Comment by JenniferYingyiWu2020** [comment]
Of course! I do need to make deep learning be integrated with RasterFrames. If the "ESRI raster function" can be used to turn that to be true, then it will be better. So, would you pls help to give me some suggestions?

**Comment by shenganzhan** [comment]
ESRI raster functions is not necessary to integrate deep learning with RasterFrames. Raster functions is a way for you to integrate that into the ArcGIS environment if you want to utilize ArcGIS's functionalities. I would suggest you to look further into the RasterFrames doc.

**Comment by JenniferYingyiWu2020** [comment]
Hi,
        I have read the “ArcGIS API for Python” and the deep learning section of “YOLOv3 Object Detector”(https://developers.arcgis.com/python/guide/yolov3-object-detector/) page and “How U-net works?”(https://developers.arcgis.com/python/guide/how-unet-works/) page have been fascinated to me. Most importantly, I think the above two pages of “Deep Learning with ArcGIS” has already met my requirements.
![001](https://user-images.githubusercontent.com/72843232/133869870-8945c696-3dc5-4fee-b2db-d21a6fcb5e13.png)
![002](https://user-images.githubusercontent.com/72843232/133869875-9c2958bd-7193-4877-9265-1e270c05f3a5.png)
![003](https://user-images.githubusercontent.com/72843232/133869876-fa760a65-47af-4170-9e2c-502bd4038cfd.png)
        Moreover, I have checked out the project “arcgis-python-api” from GitHub. However, the source codes and data sets of “14-deep-learning” have not been found, only there are some “.ipynb” files.
![004](https://user-images.githubusercontent.com/72843232/133869886-a0a46682-9e25-4176-9511-5263f376e89e.png)
        So, if I would like to use “raster functions” to integrate “YOLO object detection” and “U-Net semantic segmentation” into the ArcGIS environment, could you pls help to share the “source code” and “data sets” for me? Many Thanks!

**Comment by shenganzhan** [comment]
Again, raster function is not necessary for your purpose. You can find examples of ArcGIS API for Python here: https://github.com/Esri/arcgis-python-api/tree/master/samples/04_gis_analysts_data_scientists
